src/mappings/talent-layer-escrow.ts

```diff
@@ -80,6 +80,14 @@ export function handlePaymentCompleted(event: PaymentCompleted): void {
   const service = getOrCreateService(event.params._serviceId)
   service.status = 'Finished'
   service.save()
+
```

Comment:
had to add "!" to "service.buyer!" & "service.seller!" for the compiler.

In theory these fields cannot be null at this stage, so should not be a risk to break the graph.